sway/desktop/render.c

```diff
@@ -468,6 +813,37 @@ static void render_saved_view(struct sway_view *view, struct sway_output *output
 		scale_box(&dst_box, wlr_output->scale);
 
 		deco_data.corner_radius *= wlr_output->scale;
+
+		// Render blur
+		struct wlr_gles2_texture_attribs attribs;
+		wlr_gles2_texture_get_attribs(saved_buf->buffer->texture, &attribs);
+		if (deco_data.blur
+				&& view->container->blur_enabled
+				// Check if window has alpha
+				&& !(!attribs.has_alpha && view->container->alpha >= 1.0f && deco_data.alpha >= 1.0f)
+				&& should_parameters_blur()) {
+			int width, height;
+			wlr_output_transformed_resolution(output->wlr_output, &width, &height);
+			struct wlr_fbox src_box = {
+				.x = 0,
+				.y = 0,
+				.width = width,
+				.height = height,
+			};
+
+			pixman_region32_t opaque_region;
+			pixman_region32_init(&opaque_region);
+			pixman_region32_union_rect(&opaque_region, &opaque_region, 0, 0, 0, 0);
+
+			// Use the pre-rendered blurred buffer for tiled windows due tiled
+			// that all tiled windows are displaying the same content under the windows
+			bool is_floating = container_is_floating(view->container);
+			render_blur(!is_floating, output, damage, &src_box, &dst_box, &opaque_region,
+					saved_buf->width, saved_buf->height, 1, deco_data, config->blur_params);
+
+			pixman_region32_fini(&opaque_region);
+		}
+
```

Comment:
see above comment (would this be better suited to live in render_view? Right now it seems like we just want to blur views anyway)